src/app/core/views/portinfo/analog/index.ts

```diff
@@ -36,4 +36,22 @@ export const AnalogPortInfo: PortInfoRecord<AnalogComponent> = {
             ],
         }],
     },
+    "Inductor": {
+        ...DefaultAnalogPortInfo,
+        PositionConfigs: [{
+            "ports": [
+                { origin: V(0, -0.925), target: V(0, -1.425), dir: V(0, -1) },
+                { origin: V(0, +0.925), target: V(0, +1.425), dir: V(0, +1) },
+            ],
+        }],
+    },
+    "CurrentSource": {
+        ...DefaultAnalogPortInfo,
+        PositionConfigs: [{
+            "ports": [
+                { origin: V(0, -0.5), target: V(0, -1.3), dir: V(0, -1) },
+                { origin: V(0, +0.5), target: V(0, +1.3), dir: V(0, +1) },
+            ],
+        }],
```

Comment:
If you want to go above and beyond here, you could create a utility method in `app/core/views/portinfo/positioning/utils.ts` called like `TwoPortPositioner` which would be like:
```ts
export function TwoPortPositioner(dir: "horizontal" | "vertical", size: number): PortConfig[]
```
which would be used like
```ts
"Resistor": {
    ...
    PositionConfigs: [{ "ports": TwoPortPositioner("horizontal", 1.2) }],
}
...
"Inductor": {
    ...
    PositionConfigs: [{ "ports": TwoPortPositioner("vertical", 240/104*0.8) }],     
},
...
"CurrentSource": {
    ...
    PositionConfigs: [{ "ports": TwoPortPositioner("vertical", 1) }],
},
...
```